new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItSimpleValidation.java

```diff
@@ -0,0 +1,341 @@
+    assertNotNull(namespaces.get(0), "Namespace is null");
+    domainNamespace1 = namespaces.get(1);
+
+
+    // Create a service account for the unique namespace
+    serviceAccountName = opNamespace + "-sa";
+    V1ServiceAccount serviceAccount = assertDoesNotThrow(
+        () -> Kubernetes.createServiceAccount(new V1ServiceAccount()
+            .metadata(new V1ObjectMeta().namespace(opNamespace).name(serviceAccountName))));
+    logger.info("Created service account: {0}", serviceAccount.getMetadata().getName());
+
+    // create persistent volume and persistent volume claim
+    String pvcName = domainUid + "-pvc"; // name of the persistent volume claim
+    String pvName = domainUid + "-pv"; // name of the persistent volume
+
+    V1PersistentVolumeClaim v1pvc = new V1PersistentVolumeClaim()
+        .spec(new V1PersistentVolumeClaimSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUid + "-weblogic-domain-storage-class")
+            .volumeName(pvName)
+            .resources(new V1ResourceRequirements()
+                .putRequestsItem("storage", Quantity.fromString("10Gi"))))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvcName)
+            .withNamespace(domainNamespace1)
+            .build()
+            .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+            .putLabelsItem("weblogic.domainUid", domainUid));
+
+    boolean success = assertDoesNotThrow(
+        () -> TestActions.createPersistentVolumeClaim(v1pvc),
+        "Persistent volume claim creation failed, "
+        + "look at the above console log messages for failure reason in ApiException responsebody"
+    );
+    assertTrue(success, "PersistentVolumeClaim creation failed");
+
+    V1PersistentVolume v1pv = new V1PersistentVolume()
+        .spec(new V1PersistentVolumeSpec()
+            .addAccessModesItem("ReadWriteMany")
+            .storageClassName(domainUid + "-weblogic-domain-storage-class")
+            .volumeMode("Filesystem")
+            .putCapacityItem("storage", Quantity.fromString("10Gi"))
+            .persistentVolumeReclaimPolicy("Recycle")
+            .hostPath(new V1HostPathVolumeSource()
+                .path(TestConstants.PV_ROOT + "/" + domainUid + "-persistentVolume")))
+        .metadata(new V1ObjectMetaBuilder()
+            .withName(pvName)
+            .withNamespace(domainNamespace1)
+            .build()
+            .putLabelsItem("weblogic.resourceVersion", "domain-v2")
+            .putLabelsItem("weblogic.domainUid", domainUid));
+    success = assertDoesNotThrow(
+        () -> TestActions.createPersistentVolume(v1pv),
+        "Persistent volume creation failed, "
+        + "look at the above console log messages for failure reason in ApiException responsebody"
+    );
+    assertTrue(success, "PersistentVolume creation failed");
+  }
+
+  @Test
+  @Order(1)
+  @DisplayName("Create a MII image")
+  public void testCreatingMiiImage() {
```

Comment:
FYI  - I am removing ItWitValidation.java in my PR #1616 because the content of the class (and this method) has been moved into ItMiiDomain.java for the first end-to-end test case. So I don't think we need this test method here.

Reply:
Actually, everything in this new test class are covered in ItMiiDomain.java. We could just remove ItSimpleDomainValidation.java and ItSimpleOperatorValidation.java as well, and don't need to do the consolidation.

Reply:
As we discussed in the meeting I will remove the ItWitValidation test and keep the operator and domain tests